examples/chip-tool-darwin/commands/common/CHIPCommandStorageDelegate.mm

```diff
@@ -17,14 +17,43 @@ BOOL CHIPSetDomainValueForKey(NSString * domain, NSString * key, id value)
     return CFPreferencesAppSynchronize((CFStringRef) domain) == true;
 }
 
-void CHIPRemoveDomainValueForKey(NSString * domain, NSString * key)
+BOOL CHIPRemoveDomainValueForKey(NSString * domain, NSString * key)
```

Comment:
Should `removeStorageDataForKey` propagate this return value through?

Reply:
Yes, that makes sense. Updated.